Formula/ca-certificates.rb

```diff
@@ -17,6 +17,8 @@ class CaCertificates < Formula
 
   def install
     pkgshare.install "cacert-#{version}.pem" => "cacert.pem"
+    # Ensure we have identical bottles by making the timestamps uniform across OS versions
+    touch pkgshare/"cacert.pem", mtime: path.mtime
```

Comment:
This isn't working for some reason. Weird.

Reply:
So this is changing the `mtime` of `cacert.pem` as expected. Except `brew` is changing it back upon bottling.